### PR TITLE
Mul 209 payment status update for secondary orders

### PR DESCRIPTION
**Pull request by MacBalc**

### CODE REVIEW

**Review by rafalmal — APPROVED** [review]

File: tests/Behat/Context/Shop/OrderContext.php [hunk 1]
```diff
@@ -28,7 +32,7 @@ class OrderContext extends RawMinkContext implements Context
     public function __construct(
         ShowProductPage $productPage,
         SharedStorageInterface $sharedStorage,
-    ) {
+        ) {
```
Comment (rafalmal) [inline tests/Behat/Context/Shop/OrderContext.php:35]: this is incorrect

**Review by rafalmal — CHANGES_REQUESTED** [review]

File: tests/Behat/Context/Shop/OrderContext.php [hunk 2]
```diff
@@ -44,6 +48,21 @@ public function iShouldSeeOrders($count)
...
+     * @Then I should see :count orders with :status status
+     */
+    public function iShouldSeeOrdersWithStatus(int $count, string $status)
+    {
+        $page = $this->getSession()->getPage();
+        $tableWrapper = $page->find('css', 'table');
+        $orders = $tableWrapper->findAll('css', '.item');
+        Assert::eq(count($orders), $count);
+        $labels = $page->findAll('css', '.ui.green.label');
+        foreach ($labels as $label){
```
Comment (rafalmal) [inline tests/Behat/Context/Shop/OrderContext.php:61]:
```suggestion
        foreach ($labels as $label) {
```

**Review by rafalmal — CHANGES_REQUESTED** [review]

File: tests/Behat/Context/Shop/OrderContext.php (hunk 1 above)
Comment (MacBalc) [reply tests/Behat/Context/Shop/OrderContext.php]: done

**Review by rafalmal — APPROVED** [review]

File: tests/Behat/Context/Shop/OrderContext.php [hunk 3]
```diff
@@ -44,6 +55,21 @@ public function iShouldSeeOrders($count)
...
+    /**
+     * @Then I should see :count orders with :status status
+     */
+    public function iShouldSeeOrdersWithStatus(int $count, string $status)
+    {
+        $page = $this->getSession()->getPage();
+        $tableWrapper = $page->find('css', 'table');
+        $orders = $tableWrapper->findAll('css', '.item');
+        Assert::eq(count($orders), $count);
+        $labels = $page->findAll('css', '.ui.green.label');
```
Comment (Wojdylak) [inline tests/Behat/Context/Shop/OrderContext.php:67]: Why only `green`?

Comment (rafalmal) [reply tests/Behat/Context/Shop/OrderContext.php:67]: I check only for green labeled "complete" or "fullfilled" orders, but extended flexibility of this method in next commit.

**Pull request merged**